### PR TITLE
Update python.md

### PR DESCRIPTION
**Pull request by DenizNazari**
## What type of PR is this? (check all applicable)

- [x] Added new link to category
- [ ] New category added
- [ ] Fixed broken link
- [ ] Typo fix
- [ ] Readme update
- [ ] Optimization
- [ ] Documentation update
- [ ] Frontend update

## Description
0 dan python oyrenmek icin  notlar .

### CODE REVIEW

**Review by YusufBilgin — APPROVED** [review]
:+1:

File: docs/programming-languages/python.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ sidebar_label: Python
 - [Tons of Algorithmic Solutions with Python](https://github.com/TheAlgorithms/Python)
 - [Python Projects with Beginners](https://beginnerpythonprojects.com)
 - [Python Programlama öğreten oyun  | checkio](https://py.checkio.org/ "Python Programlama öğreten oyun")
+- [Python Programlama notlari|notlar](https://github.com/e-k-eyupoglu/python_tutorial/tree/main/main)
```
Comment (damla) [inline docs/programming-languages/python.md:20]:
```suggestion
- [CSD-1993/KursNotlari|uyqulamali notlar](https://github.com/CSD-1993/KursNotlari/blob/master/Python-App.pdf)
- [Python Programlama notlari|notlar](https://github.com/e-k-eyupoglu/python_tutorial/tree/main/main)
```

Comment (damla) [inline docs/programming-languages/python.md:21]:
```suggestion
- [Python Programlama notlari | notlar](https://github.com/e-k-eyupoglu/python_tutorial/tree/main/main)
```

**Review by damla — COMMENTED** [review]
Typo fix will be added.

File: docs/programming-languages/python.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ sidebar_label: Python
 - [Tons of Algorithmic Solutions with Python](https://github.com/TheAlgorithms/Python)
 - [Python Projects with Beginners](https://beginnerpythonprojects.com)
 - [Python Programlama öğreten oyun  | checkio](https://py.checkio.org/ "Python Programlama öğreten oyun")
+- [CSD-1993/KursNotlari|uyqulamali notlar](https://github.com/CSD-1993/KursNotlari/blob/master/Python-App.pdf)
+- [Python Programlama notlari | notlar](https://github.com/e-k-eyupoglu/python_tutorial/tree/main/main)
```
Comment (damla) [inline docs/programming-languages/python.md:21]:
```suggestion
- [CSD-1993 / Kurs Notlari | Uygulamalı Notlar](https://github.com/CSD-1993/KursNotlari/blob/master/Python-App.pdf)
- [Python Programlama Notları](https://github.com/e-k-eyupoglu/python_tutorial/tree/main/main)
```

**Review by damla — APPROVED** [review]
Fixed 👍🏼🚢

**Pull request merged**